tests/eosio.system_tests.cpp

```diff
@@ -4743,38 +4783,52 @@ BOOST_FIXTURE_TEST_CASE( update_rex, eosio_system_tester, * boost::unit_test::to
                        == get_producer_info(producer_names[20])["total_votes"].as<double>() );
 
    BOOST_REQUIRE_EQUAL( success(), updaterex( alice ) );
-   produce_block( fc::days(20) );
+
+   produce_blocks( 1 );
+   produce_block( fc::days(10) );
+   produce_blocks( 1 );
+
    BOOST_TEST_REQUIRE( get_producer_info(producer_names[20])["total_votes"].as<double>()
                        < stake2votes( asset( get_voter_info( alice )["staked"].as<int64_t>(), symbol{CORE_SYM} ) ) );
+
    BOOST_REQUIRE_EQUAL( success(), updaterex( alice ) );
    BOOST_TEST_REQUIRE( stake2votes( asset( get_voter_info( alice )["staked"].as<int64_t>(), symbol{CORE_SYM} ) )
                        == get_producer_info(producer_names[20])["total_votes"].as<double>() );
 
+   produce_blocks( 1 );
+   produce_block( fc::hours(19 * 24 + 23) );
+   produce_blocks( 1 );
```

Comment:
Same thing here. This version of the tests have the additional `produce_blocks` while the one in #407 does not.